pkg/stargate/stargate.go

```diff
@@ -45,19 +46,27 @@ func NewStargate(
 		cassandraEncryption.ServerEncryptionStores = kc.Spec.Cassandra.ServerEncryptionStores
 	}
 
+	labels := map[string]string{
```

Comment:
Could we extract this to a new `createResourceMeta` function like you did in some other places?